app/controllers/application_controller.rb

```diff
@@ -2,12 +2,21 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   before_action :find_user
+  # before_action :require_login
 
```

Comment:
I love the idea of requiring login as a controller filter, but you've commented this line out!

Remember, you can disable this filter for specific actions using `skip_before_action`.